tests/sentry/db/test_router.py

```diff
@@ -24,6 +24,11 @@ def test_for_control(self):
         assert router.allow_migrate("control", "sentry", User)
         assert not router.allow_migrate("region", "sentry", User)
 
+        # Ensure tables that no longer exist don't fail
+        assert router.allow_migrate(
```

Comment:
We could ditch this when we squash migrations, right?

Reply:
I don't think so. We'll likely drop more tables in the future.